### PR TITLE
OCPBUGS-39084: fix pod preview error

### PR DESCRIPTION
**Pull request by upalatucci**
fix pop preview .

On loading the TreeView was raising an error. Just show the loading icon.

when pods length > maxPreviewPods , show `Showing {{shown}} from {{total}} results` an otherwise `'View all {{total}} results`  

The current classnames don't give the expected style so we need to copy the CSS. 
Even importing the patternfly CSS file (@patternfly/react-style/src/components) in react doesn't work. 

Removing stylelint as it was creating lots of issues for the CSS file . We don't use stylelint anywhere and scss is formatted by prettier. don't make sense to have stylelint as we ond't use css

**After**

Screenshot 2024-08-29 at 15 51 45

### CODE REVIEW

File: src/views/networkpolicies/new/components/NetworkPolicySelectorPreview/components/NetworkPolicyPodsPreview.tsx [hunk 1]
```diff
@@ -119,22 +124,27 @@ const NetworkPolicyPodsPreview: FC<NetworkPolicyPodsPreviewProps> = ({
           t('List of pods')
         )}
       </div>
-      <TreeView data={preview?.pods} data-test="pods-preview-tree" hasGuides />
+      <TreeView
+        className="network-policy-pods-preview"
+        data={preview.pods}
```
Comment (metalice) [inline src/views/networkpolicies/new/components/NetworkPolicySelectorPreview/components/NetworkPolicyPodsPreview.tsx:129]: u removed the ? form .pods

**Review by avivtur — APPROVED** [review]

**Pull request merged**